accounting/filters/OsgScheddCpuRemovedFilter.py

```diff
@@ -0,0 +1,400 @@
+        badput_cpu_time = []
+        total_cpu_time = []
+        for (goodput_time, badput_time, total_time, cpus) in zip(
+                data["CommittedTime"],
+                data["_BadWallClockTime"],
+                data["RemoteWallClockTime"],
+                data["RequestCpus"]):
+            if None in [goodput_time, cpus]:
+                goodput_cpu_time.append(None)
+            else:
+                goodput_cpu_time.append(goodput_time * cpus)
+            badput_cpu_time.append(badput_time * cpus)
+            if None in [total_time, cpus]:
+                total_cpu_time.append(None)
+            else:
+                total_cpu_time.append(total_time * cpus)
+
+        # Don't count starts and shadows for jobs that don't/shouldn't have shadows
+        num_exec_attempts = []
+        num_shadow_starts = []
+        for (job_starts, shadow_starts, no_shadow) in zip(
+                data["NumJobStarts"],
+                data["NumShadowStarts"],
+                data["_NoShadow"]):
+            if no_shadow:
+                num_exec_attempts.append(None)
+                num_shadow_starts.append(None)
+            else:
+                num_exec_attempts.append(job_starts)
+                num_shadow_starts.append(shadow_starts)
+
+        # Short jobs are jobs that ran for < 1 minute
+        is_short_job = []
+        for (goodput_time, record_date, start_date) in zip(
+                data["CommittedTime"],
+                data["RecordTime"],
+                data["JobCurrentStartDate"]):
+            if (goodput_time is not None) and (goodput_time > 0):
+                is_short_job.append(goodput_time < 60)
+            elif None in (record_date, start_date):
+                is_short_job.append(None)
+            else:
+                is_short_job.append((record_date - start_date) < 60)
+
+        # "Long" (i.e. "normal") jobs ran >= 1 minute
+        # We only want to use these when computing percentiles,
+        # so filter out short jobs and removed jobs,
+        # and sort them so we can easily grab the percentiles later
+        long_times_sorted = []
+        for (is_short, goodput_time, no_shadow, job_status) in zip(
+                is_short_job,
+                data["CommittedTime"],
+                data["_NoShadow"],
+                data["JobStatus"]):
+            if (is_short == False) and (no_shadow == False) and (job_status != 3):
+                long_times_sorted.append(goodput_time)
+        long_times_sorted = self.clean(long_times_sorted)
+        long_times_sorted.sort()
+
+        # Compute columns
+        row["All CPU Hours"]    = sum(self.clean(total_cpu_time)) / 3600
+        row["Good CPU Hours"]   = sum(self.clean(goodput_cpu_time)) / 3600
+        row["Num Uniq Job Ids"] = sum(data['_NumJobs'])
+        row["Num Rm'd Jobs"]    = sum([status == 3 for status in data["JobStatus"]])
+        row["Num Jobs Rm'd w/o exec"]= sum([starts == 0 for starts in data["NumJobStarts"]]) 
```

Comment:
Following suggested changes above
```suggestion
        row["Num Rm'd Jobs w/o Execs"]= sum([starts == 0 for starts in data["NumJobStarts"]]) 
```